daemon/logger/alilogs/logapi.go

```diff
@@ -0,0 +1,72 @@
+// Pakcage alilogs api interface
+
+package alilogs
+
+import (
+	"errors"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/galaxydi/go-loghub"
+)
+
+// AliLogAPI define log api interface
+type AliLogAPI interface {
+	PutLogs(*sls.LogGroup) error
+}
+
+// AliLogClient implements AliLogAPI interface
+type AliLogClient struct {
+	Endpoint        string
+	ProjectName     string
+	LogstoreName    string
+	accessKeyID     string
+	accessKeySecret string
+	sessionToken    string
+	project         *sls.LogProject
```

Comment:
Seems these 4 vars (id, secret, token, project) don't have to be part of the client struct. They are used to construct logstore and then can be thrown away.